lib/net/peer.js

```diff
@@ -203,6 +204,28 @@ class Peer extends EventEmitter {
     return this.address.hostname;
   }
 
+  /**
+   * Getter to retrieve fullname.
+   * @returns {String}
+   */
+
+  fullname() {
+    return this.address.fullname();
+  }
+
+  /**
+   * Getter to retrieve local fullname.
+   * Pull the key from this.address to
+   * prevent storing the key twice
+   * @returns {String}
+   */
+
+  localname() {
+    const {hostname, port} = this.local;
+    const key = this.address.getKey();
+    return IP.toHostname(hostname, port, key);
```

Comment:
Look at the implementation of `IP.toHostName` to see if its simple enough to use here so that there is no need to include `binet`